Fix typos in README.md (#48)
0xF6/mana_lang: README.md

cp -R ./backend/mana.backend.ishtar.light/bin/net6.0/win10-x64/native/ ./output
```

The `output` folder should contain:
- ishtar.exe - main ishtar file
- ishtar.exp - export metadata for main module
- ishtar.lib - dynamic library for main module
- ishtar.pdb - debug symbols

> #### Compile manяc

Reviewer: ```suggestion
#### Compile manac
```

/test: folder with various tests
```

You can run all tests from the root directory with `dotnet test`.

To recompile the vm and the compiler: `dotnet build`.

To recompile the standard library: `manac ./mana.std/corlib.wproj`.

After your changes are done, please remember to run `dotnet format` to guarantee all files are properly formatted and
then run the full suite with `dotnet test`.

## Q\A

```yaml
Q:
  Why is it called mana?
A:
  I liked it very much 🗿🗿🗿
  So, I tried to choose a memorable name that would be easy
  simple for the tools (like the 'rune' package manager).

>   The original name was Wave Lang, but I didn't like it, and it was chosen by a chat in Telegram 🙂.

Reviewer: ```suggestion
  The original name was Wave Lang, but I didn't like it, and it was chosen by random.
```

Q:
  Why it based on C#?
A:
  Initially, i started developing a virtual machine in C++,
  but there were a lot of difficulties with basic things (such as collections, text formatting, etc.)
  And at some point i saw that microsoft began to develop a fully AOT compiler for dotnet.
  That means we could write in pure C# without using runtime and std, which allows everyone to write such hard things like an OS!
  So I decided - that's it! I'm Definitely writing a virtual machine in C#!

  So, now I'm developing using the C# runtime and the std, but in version 2.0 I'm planning to completely move away from runtime dependencies.

Q:
  This language really support quantum computing?
A:

>   Not now, but in future I'm planning to add support for Microsoft Quantum Simulator, next - support for Azure Qunatum or IBM quantum cloud.

Reviewer: ```suggestion
  Not now, but in future I'm planning to add supportMicrosoft Quantum Simulator, next - support Azure Qunatum or IBM quantum cloud.
```
